draft-ietf-tls-tls13.md

```diff
@@ -3827,6 +3896,7 @@ is listed below:
 | key_share [[this document]]              |         Yes |     Clear |
 | pre_shared_key [[this document]]         |         Yes |     Clear |
 | early_data [[this document]]             |         Yes |     Clear |
+| 
```

Comment:
Actually, it looks like a blank table line which should instead list the new ticket age extension.


On Tuesday, May 03, 2016 10:17:48 pm Martin Thomson wrote:
> > @@ -3827,6 +3896,7 @@ is listed below:
> >  | key_share [[this document]]              |         Yes |     Clear |
> >  | pre_shared_key [[this document]]         |         Yes |     Clear |
> >  | early_data [[this document]]             |         Yes |     Clear |
> > +| 
> 
> Extra '|'